### PR TITLE
[MI-3414]: Fix build errors and sync with branch [mm-v6]

### PR DESCRIPTION
**Pull request by muazzamqazi**

### CODE REVIEW

File: src/components/Input/Input.d.ts [hunk 1]
```diff
@@ -1,122 +1,147 @@
...
 	 */
-    autoFocus?: boolean;
+	autoFocus?: boolean;
 
-    /**
+	/**
 	 * If `true`, the input will indicate an error
 	 *
-	 * @default false
+	 * @default ''
```
Comment (ayusht2810) [inline src/components/Input/Input.d.ts:52]: i think we should update the above comment as well

File: src/components/Input/Input.d.ts [hunk 2]
```diff
@@ -74,13 +74,6 @@ export interface InputProps {
 	 */
     className?: string;
 
-    /**
-	 * Type of the input element
-	 *
-	 * @default 'text'
-	 */
-    type?: string;
```
Comment (ayusht2810) [inline src/components/Input/Input.d.ts:82]: why this change?

File: src/components/TextArea/TextArea.stories.tsx [hunk 3]
```diff
@@ -51,3 +63,9 @@ export const AutoFocus = TextAreaTemplate.bind({});
 AutoFocus.args = {
     autoFocus: true,
 };
+
+// Custom umber of rows to occupy
```
Comment (ayusht2810) [inline src/components/TextArea/TextArea.stories.tsx:67]: fix comment

File: src/components/TextArea/TextArea.styles.ts [hunk 4]
```diff
@@ -59,6 +58,14 @@ export const StyledTextArea = styled(FormControl)({
     },
 });
 
+/**
+ * Value for
```
Comment (ayusht2810) [inline src/components/TextArea/TextArea.styles.ts:62]: is this completed?

File: src/components/List/List.component.tsx [hunk 5]
```diff
@@ -5,6 +5,7 @@ import {extendClassname} from '@Utils';
 
 import {ListPropType} from './List';
 import {Options} from './List.styles';
+import { Icon } from '@Components/Icon';
```
Comment (ayusht2810) [inline src/components/List/List.component.tsx:8]: why there are mismatches in spaces around brackets?

File: src/components/Input/Input.d.ts (hunk 2 above)
Comment (muazzamqazi) [reply src/components/Input/Input.d.ts:82]: `type` is included in the `FormControlProps` extended above

**Review by ayusht2810 — APPROVED** [review]

File: src/components/Input/Input.stories.tsx [hunk 6]
```diff
@@ -17,10 +17,17 @@ Default.args = {
     label: 'Label',
 };
 
+// Large size Default
```
Comment (avas27JTG) [inline src/components/Input/Input.stories.tsx:20]: make it descriptive

**Review by avas27JTG — APPROVED** [review]
